conf/logback.xml

```diff
@@ -35,14 +22,6 @@
     </appender>
 
 
-    <logger name="accesslog" level="INFO" additivity="false">
-        <appender-ref ref="ACCESS_LOG_FILE" />
-    </logger>
-
-    <logger name="com.ning.http.client.providers.netty" additivity="false">
-        <appender-ref ref="STDOUT_IGNORE_NETTY" />
-    </logger>
-
     <logger name="uk.gov" level="INFO"/>
 
     <logger name="org.asynchttpclient.netty" level="INFO"/>
```

Comment:
This entry can be removed:
logger name="org.asynchttpclient.netty" level="INFO"

Reply:
Removed